test/api/widgets.test.js

```diff
@@ -1,25 +1,99 @@
 import 'test/enzyme-init';
-import getWidget from 'api/widget';
-import { BODY_OK, BODY_ERROR } from 'test/mocks/widget';
+import { getWidget, getWidgets } from 'api/widgets';
+import { makeRequest, METHODS } from 'api/apiManager';
+import { BODY_OK, WIDGET_LIST } from 'test/mocks/widgets';
 
-const WIDGET_CODE = 'test_widget';
+const correctRequest = {
+  uri: '/api/widgets',
+  method: METHODS.GET,
+  mockResponse: WIDGET_LIST.payload,
+  useAuthentication: true,
+  errors: expect.any(Function),
+};
 
-jest.unmock('api/widget');
+const correctRequestWidget = {
```

Comment:
You can omit mockResponse and errors (these props are not relevant to the test). In the test, you can use objectContaining, like:

```javascript
expect(makeRequest).toHaveBeenCalledWith(expect.objectContaining({
        uri: `/api/pages/${CONTACTS_PAYLOAD.code}`,
        body: CONTACTS_PAYLOAD,
        method: METHODS.PUT,
        useAuthentication: true,
      }));
```